training/tensorboard/pytorch/pytorch_smdataparallel_mnist_demo.ipynb

```diff
@@ -0,0 +1,811 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Estimator function options\n",
+    "\n",
+    "To instruct SageMaker to find the local path where the model tensors are saved and upload the tensor data to an Amazon S3 bucket, configure the `sagemaker.debugger.TensorBoardOutputConfig` class as follows.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.debugger import TensorBoardOutputConfig\n",
+    "\n",
+    "tensorboard_s3_output_path = \"s3://{}/{}\".format(bucket, prefix)\n",
+    "\n",
+    "# Create a TensorBoardOutputConfig object to configure automatic data upload from\n",
+    "# the training job's local data storage to S3\n",
+    "tensorboard_output_config = TensorBoardOutputConfig(\n",
+    "    s3_output_path=tensorboard_s3_output_path,\n",
+    "    container_local_output_path=\"/opt/ml/output/tensorboard\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.pytorch import PyTorch\n",
+    "\n",
+    "env = {\n",
+    "    \"SAGEMAKER_REQUIREMENTS\": \"requirements.txt\",  # tensorboard dependency required for PyTorch\n",
+    "}\n",
+    "\n",
+    "estimator = PyTorch(\n",
+    "    base_job_name=\"pytorch-tensorboard-dataparallel-mnist\",\n",
+    "    source_dir=\"code\",\n",
+    "    entry_point=\"train_pytorch_smdataparallel_mnist.py\",\n",
+    "    role=role,\n",
+    "    framework_version=\"1.11.0\",\n",
+    "    py_version=\"py38\",\n",
+    "    instance_count=1,\n",
+    "    env=env,\n",
+    "    instance_type=\"ml.p3.16xlarge\",\n",
+    "    sagemaker_session=sagemaker_session,\n",
+    "    distribution={\"smdistributed\": {\"dataparallel\": {\"enabled\": True}}},\n",
+    "    # Configure TensorBoard output config in estimator to enable automatic sync in training job\n",
+    "    tensorboard_output_config=tensorboard_output_config,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator.fit(wait=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Track training job in TensorBoard\n",
+    "\n",
+    "Once the job is started, you can generate the URL to the TensorBoard application and add your training job directly into the TensorBoard application and retrieve a link by using the `TensorBoardApp().get_app_url()` API from `agemaker.interactive_apps.tensorboard`.\n",
+    "\n",
+    "If you are running the command from SageMaker Studio, the Python SDK will generate a direct URL. Otherwise, you will receive an AWS console URL, which you can access in order to select the application Domain and User Profile that you want the TensorBoard session to run under. \n",
```

Comment:
```suggestion
    "If you are running this notebook in SageMaker Studio, the `TensorBoardApp().get_app_url()` API will generate a direct URL. Otherwise, you will receive a URL to the Amazon SageMaker console, which you can access SageMaker Domain and then the Domain user profile under which you want the TensorBoard session to run. \n",
```